tools/scripts/ValidateReleaseInfo.ps1

```diff
@@ -74,8 +74,15 @@ function ValidateChannelInfo($channel)
 	return $isCurrentVersionValid -and $isMinimumVersionValid -and $isVersionRelationshipValid -and $isInstallerUrlValid -and $isReleaseNotesUrlValid
 }
 
+# Determine our octokit version
+$octokitVersion = $Env:OctokitVersion
```

Comment:
This is set to 0.33.0 in the build definition for the PR build. Having the default value here is mainly for local use